src/core/components/AppSimpleDialog.vue

```diff
@@ -0,0 +1,75 @@
+<template>
```

Comment:
I believe theres already a dialog component

Reply:
I can't find popup or dialog component like this anywhere.